news/_posts/2020-06-18-about-bc-breaks.md

```diff
@@ -0,0 +1,398 @@
+```
+
+When you release a new version of your software, if the new version only contains bug fixes, it is a patch version and you increase the PATCH number.
+
+> Example: 3.5.27 => 3.5.28
+
+When you release a new version of your software, if the new version only contains bug fixes and new features, but everything that was working before still works, it is a minor version and you increase the MINOR number and reset the PATCH.
+
+> Example: 3.5.27 => 3.6.0
+
+When you release a new version of your software, if the new version number contains code changes that might break other software relying on it, it is a major version and you increase the MAJOR number and reset the MINOR and the PATCH numbers.
+
+> Example: 3.5.27 => 4.0.0
+
+<hr>
+
+As a developer who builds software relying on this project, when you see a new version number, you can see whether this is a patch, a minor or a major version. And consequently you know that:
+
+- If it's a patch version, upgrading to use the new version will bring no change in behavior, it will only bring bug fixes
+- If it's a minor version, upgrading to use the new version might bring new behaviors you can use but all behaviors you are using are still usable
+- If it's a major version, you need to check what has been changed to see whether you need to adjust your code
+
+This is very useful for developers. Being able, from the version number, to understand the impact this change can have on your software is very useful!
+
+### Why do projects introduce BC breaks ?
+
+When reading this, you might wonder why people release new major versions of a software. If everybody would only release minor versions following SemVer, nobody would ever be afraid of upgrading its software because of BC breaks! Would'nt it be amazing, a software world without BC breaks and the fear of them?
+
+Moreover PrestaShop is a CMS, which means it is customizable and extensible. Extension often comes from installing modules and a theme.
+
+These modules and themes are built by developers that want to sell it to an audience as large as possible. A BC break that impacts some code they are using is [a bother to them](https://www.snoyman.com/blog/2018/04/stop-breaking-compatibility).
+It means for example they have to provide a different version of their product for PrestaShop 1.7.5 and 1.7.6 because of a change in PrestaShop 1.7.6.
+
+BC breaks hinder the capability of modules to provide a huge compatibility range.
+
+Unfortunately... BC breaks are necessary for software to evolve. Not all software changes can be done while maintaining backward compatibility, and BC breaks are a necessary step for a project to evolve in time. And [evolution is necessary](https://build.prestashop.com/news/prestashop-in-2019-and-beyond-part-3-the-future-architecture/) to ensure it continues to fit the needs of its user community, who evolve in time. A software that does not evolve is bound to disappear, replaced by better contributors.
```

Comment:
I notice you wrote "a software" at several places, but software is an uncountable word, for example here you'd write 

```suggestion
Unfortunately... BC breaks are necessary for software to evolve. Not all software changes can be done while maintaining backward compatibility, and BC breaks are a necessary step for a project to evolve in time. And [evolution is necessary](https://build.prestashop.com/news/prestashop-in-2019-and-beyond-part-3-the-future-architecture/) to ensure it continues to fit the needs of its user community, who evolve in time. Software that does not evolve is bound to disappear, replaced by better contributors.
```

Or `A software solution`, or `A software's codebase`. 

This is truly nitpicking :D